summernote/summernote
# Table popover - add/remove columns & rows feature

#### What does this PR do?

Create a new popover on table element that allow you to:
- create new colum / row
- delete column / row
#### Where should the reviewer start?
- start on the src/summernote.js
#### How should this be manually tested?
- create a table, click on any cell. A new popover should appear to add / remove column / row
#### What are the relevant tickets?

based on pull request #1311 - that needed a rebase and rework
#### Screenshots (if for frontend)

![image](https://cloud.githubusercontent.com/assets/7001183/16208440/71976b3a-3732-11e6-9615-8fdb80cc92d3.png)
